application/helpers/mpdf/composer.json

```
@@ -24,7 +24,7 @@
 	},
 
 	"require": {
-		"php": ">=5.4.0",
+		"php": "^5.4.0 || 7.0.*",
```

```suggestion
		"php": "^5.4.0 || ^7.0.0",
```

Deixar versão PHP 7 como mínimo, pois do jeito que está não iria deixar rodar composer em PHP 7.1 e 7.2

Alias PHP 7.0 já perdeu suporte: https://www.php.net/supported-versions.php